src/components/atoms/ClickableIcon.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from "styled-components";
+import {Link} from "gatsby";
+
+const Icon = styled.img`
+    height: 25px;
```

Comment:
Is it possible to adjust this and other formats to 2 spaces and not 4? Would be great to keep spacing consistent to the rest of the code! 

```suggestion
const Icon = styled.img`
  height: 25px;
```